### PR TITLE
Add Project Details Popup

### PR DESCRIPTION
**Pull request by MrBrN197**
### Changes 
- Added desktop version popup
- Added desktop version popup
- Added project images for each project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,9 +87,9 @@
     </div>
     <div class="desktop-header">
       <nav>
-        <a href="#recent-works-start">Portfolio</a>
-        <a href="#about-start">About</a>
-        <a href="#contact-start">Contact</a>
+        <div id="recent-tag" class="link"><a href="#recent-works-start">Portfolio</a><div class="underline"></div></div>
```
Comment (SajjadAhmad14) [inline index.html:90]: The project details section on the desktop view has a bit more height on the design page. Kindly see the design and make this section exactly the same as in the design.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Orora Orora, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is followed.
- [ ] README is professional.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (MrBrN197) [reply index.html:90]: Hello I'm not sure what you mean by the desktop view has a bit more. height. on my computer it seems to have the correct display.

**Review by karvel-code — APPROVED** [review]
Hi @MrBrN197 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**